README.md

```diff
@@ -65,16 +61,26 @@ set -g theme_display_git_untracked no
 set -g theme_display_git_ahead_verbose yes
 set -g theme_display_hg yes
 set -g theme_display_virtualenv no
-set -g theme_display_ruby no
+set -g theme_display_ruby no|auto
 set -g theme_display_user yes
 set -g theme_title_display_process yes
 set -g theme_title_display_path no
+set -g theme_title_use_abbreviated_path yes
```

Comment:
change this to `no` (the rest of them are showing how to override; default here is `yes` so we need to swap this around)